src/peergos/gwt/emu/java/nio/file/Path.java

```diff
@@ -43,6 +43,18 @@ public Path resolve(String other) {
         return new Path(pathString + "/" + other);
     }
 
+    public Path resolve(Path other) {
```

Comment:
Could possibly check for null here and in a few other places in this class.

Reply:
a null pointer is fine I think